Tasks/VsTest/vstest.ts

```diff
@@ -740,7 +750,11 @@ function invokeVSTest(testResultsDirectory: string): Q.Promise<number> {
                 if ((sysDebug !== undefined && sysDebug.toLowerCase() === "true") || tiaConfig.tiaEnabled) {
                     vsTestVersionForTIA = getVsTestVersion();
 
-                    if (tiaConfig.tiaEnabled && (vsTestVersionForTIA === null || (vsTestVersionForTIA[0] < 15 || (vsTestVersionForTIA[0] === 15 && vsTestVersionForTIA[1] === 0 && vsTestVersionForTIA[2] < 25727)))) {
+                    if (tiaConfig.tiaEnabled && (vsTestVersionForTIA === null || 
+                       (vsTestVersionForTIA[0] < 14 || 
+                       (vsTestVersionForTIA[0] === 15 && vsTestVersionForTIA[1] === 0 && vsTestVersionForTIA[2] < 25727) || 
```

Comment:
We have packaged both the collector and vstest.console, what other changes which is not in < vs14 u3?